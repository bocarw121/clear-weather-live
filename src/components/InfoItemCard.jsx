import React from "react";
import { StyleSheet, View } from "react-native";

import colors from "../constants/colors";

import { dimension } from "../constants/dimension";

const InfoItemCard = ({ children }) => {
  const { infoContainer } = dimension();
  return <View style={[styles.container, infoContainer]}>{children}</View>;
};

export default InfoItemCard;

const styles = StyleSheet.create({
  container: {
    borderColor: colors.white,
    backgroundColor: colors.linearGradient[0],
    borderWidth: 2,
    borderRadius: 8,
    shadowColor: "#000",
    shadowOffset: {
      width: 0,
      height: 9,
    },
    shadowOpacity: 0.48,
    shadowRadius: 11.95,
    elevation: 18,
    margin: 10,
  },
});
