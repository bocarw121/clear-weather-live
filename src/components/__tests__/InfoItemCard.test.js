import React from "react";

import { render } from "@testing-library/react-native";

import InfoItemCard from "../InfoItemCard";

it("Should render the InfoItemCard", () => {
  const screen = render(<InfoItemCard />);

  expect(screen).toMatchSnapshot();
});
