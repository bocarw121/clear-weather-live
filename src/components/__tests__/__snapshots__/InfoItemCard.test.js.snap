// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the InfoItemCard 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgb(0, 132, 255)",
        "borderColor": "#fff",
        "borderRadius": 8,
        "borderWidth": 2,
        "elevation": 18,
        "margin": 10,
        "shadowColor": "#000",
        "shadowOffset": Object {
          "height": 9,
          "width": 0,
        },
        "shadowOpacity": 0.48,
        "shadowRadius": 11.95,
      },
      Object {
        "height": 166.75,
        "width": 312.5,
      },
    ]
  }
/>
`;
